set.mm

```diff
@@ -39333,6 +39333,20 @@ technically classes despite morally (and provably) being sets, like ` 1 `
       ACDJUEUGUFUIUGUEADCIKUFBCDHUHBCDRBDCILMNO $.
   $}
 
+  ${
+    $( The class abstraction with a conjunction is a subset of the class
+       abstraction with the left conjunct only.  (Contributed by AV,
+       7-Aug-2024.)  (Proof shortened by SN, 22-Aug-2024.) $)
+    abanssl $p |- { f | ( ph /\ ps ) } C_ { f | ph } $=
+      ( wa simpl ss2abi ) ABDACABEF $.
+
+    $( The class abstraction with a conjunction is a subset of the class
+       abstraction with the right conjunct only.  (Contributed by AV,
+       7-Aug-2024.)  (Proof shortened by SN, 22-Aug-2024.) $)
+    abanssr $p |- { f | ( ph /\ ps ) } C_ { f | ps } $=
+      ( wa simpr ss2abi ) ABDBCABEF $.
+  $}
```

Comment:
Have you try to minmize other theorems with these two ?

As for the comments, I would write **A** class... of the class....

But actually, it sounds weird to say that { x | ph } is the class abstraction **with** phi. To me, a better term is "the extension of phi". But this is a wider remark.

This remark is non-blocking.

Reply:
> Have you tried to minmize other theorems with these two ?

I just checked it: 17 proofs can be shortened using these two new theorems. I will do that in my next PR.

> As for the comments, I would write: **A** class... of the class....

Good suggestion, I will fix that.

> But actually, it sounds weird to say that { x | ph } is the class abstraction **with** phi. To me, a better term is "the extension of phi". But this is a wider remark.

I do not know how to name the wff within a class abstraction best. But the term "class abstraction" should remain in the comment.